components/AppBar/tests/unit/MDCAppBarNavigationControllerTests.swift

```diff
@@ -25,7 +25,7 @@ private class MockAppBarNavigationControllerDelegate:
   }
 }
 
-class AppBarNavigationControllerTests: XCTestCase {
+class MDCAppBarNavigationControllerTests: XCTestCase {
```

Comment:
is this change intended?

Reply:
Yeah. Xcode doesn't find associated files (like tests) when they don't match the source file name.  I don't know why we have a habit of mismatching file names, but I fixed this along the way.